node/pkg/websocketfetcher/common/type.go

```diff
@@ -14,7 +16,9 @@ const (
 	GetAllWebsocketFeedsQuery = `SELECT *
 	FROM feeds
 	WHERE definition @> '{"type": "wss"}';`
-	GetAllProxiesQuery = `SELECT * FROM proxies`
+	GetAllProxiesQuery  = `SELECT * FROM proxies`
+	VolumeCacheLifespan = 10 * time.Minute
+	VolumeFetchInterval = 10000
```

Comment:
@nick-bisonai Gemini volume fetching has sleep for 1 seconds. Did you think about what would happen if we have more than 10 Gemini symbols?

Reply:
I see the problem, I think I can use timer instead of ticker